tools/release/d8-pull.sh

```diff
@@ -264,7 +264,9 @@ if [[ "$PULL_RELEASE_METADATA_IMAGES" == "yes" ]]; then
 fi
 
 #pull trivy CVE database
-pull_trivy_db
+if [[ "$EDITION" != "ce" ]]; then
+  pull_trivy_db
```

Comment:
@alex123012 We have problems with skopeo in pull_trivy_db. Please, check can we use crane instead of skopeo for trivy db images.